### PR TITLE
Hw 27 2

### PR DESCRIPTION
**Pull request by IrinaDyakonova**

### CODE REVIEW

File: src/main/java/mate/academy/cinemaproject/util/HashUtil.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        SecureRandom random = new SecureRandom();
+        byte[] salt = new byte[16];
+        random.nextBytes(salt);
+        return salt;
+    }
+
+    public static String hashPassword(String password, byte[] salt) {
+        StringBuilder hashedPassword = new StringBuilder();
+        try {
+            MessageDigest messageDigest = MessageDigest.getInstance("SHA-512");
```
Comment (boroda4436) [inline src/main/java/mate/academy/cinemaproject/util/HashUtil.java:21]: make SHA-512 as a constant

File: src/main/java/mate/academy/cinemaproject/util/HashUtil.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            messageDigest.update(salt);
+            byte[] digest = messageDigest.digest(password.getBytes());
+            for (byte b: digest) {
+                hashedPassword.append(String.format("%02x",b));
+            }
+        } catch (NoSuchAlgorithmException e) {
+            LOGGER.error("Can't hash a password ", e);
```
Comment (boroda4436) [inline src/main/java/mate/academy/cinemaproject/util/HashUtil.java:28]: throw exception?

Comment (IrinaDyakonova) [reply src/main/java/mate/academy/cinemaproject/util/HashUtil.java:28]: done

File: src/main/java/mate/academy/cinemaproject/util/HashUtil.java (hunk 1 above)
Comment (IrinaDyakonova) [reply src/main/java/mate/academy/cinemaproject/util/HashUtil.java:21]: done

**Pull request merged**